src/app/team/page.tsx

```diff
@@ -0,0 +1,66 @@
+"use client"
+
+import Card, { MemberCard } from '~/components/Card';
+import data from "../../../public/team.json";
+import Marquee from 'react-fast-marquee';
+import { useState } from 'react';
+
+
+export default function Team() {
+    const [team, setTeam] = useState<1|2>(1);
+     return (
+        <div className='min-h-screen'>
```

Comment:
![image](https://github.com/user-attachments/assets/da1894fd-bdfe-4bf7-8d44-e0b4f68e79e0)

content underflow in small screens